docs/user_guides/fs/transformation_functions.md

```diff
@@ -0,0 +1,163 @@
+
+=== "Python"
+    !!! example "Creation of a one-to-many transformation function in Hopsworks."
+        ```python
+        from hopsworks import udf
+        import pandas as pd
+
+        @udf(return_type=[int, int])
+        def add_one_and_two(feature1):
+            return pd.DataFrame({"add_one":feature1 + 1, "add_two":feature1 + 2})
+        ```
+
+### Many-to-many transformations
+
+The creation of a many-to-many transformation function is similar to that of a one-to-many transformation function, the only difference being that the transformation function accepts multiple features as input.
+
+=== "Python"    
+    !!! example "Creation of a many-to-many transformation function in Hopsworks."
+        ```python
+        from hopsworks import udf
+        import pandas as pd
+
+        @udf(return_type=[int, int, int])
+        def add_one_multiple(feature1, feature2, feature3):
+            return pd.DataFrame({"add_one_feature1":feature1 + 1, "add_one_feature2":feature2 + 1, "add_one_feature3":feature3 + 1})
+        ```
+
+### Dropping input features
+
+The `drop` parameter of the `@udf` decorator is used to drop specific features after transformation.  If any argument of the transformation function is passed to the `drop` parameter, then the feature mapped to the argument is dropped after the transformation functions are applied. In the example below, the features mapped to the arguments `feature1` and `feature2` are dropped after the application of all transformation functions.
+
+
+=== "Python"    
+    !!! example "Specify arguments to drop after transformation"
+        ```python
+        from hopsworks import udf
+        import pandas as pd
+
+        @udf(return_type=[int, int, int], drop=["feature1", "feature3"])
+        def add_one_multiple(feature1, feature2, feature3):
+            return pd.DataFrame({"add_one_feature1":feature1 + 1, "add_one_feature2":feature2 + 1, "add_one_feature3":feature3 + 1})
+        ```
+
+### Training dataset statistics
+
+A keyword argument `statistics` can be defined in the transformation function if it requires training dataset statistics for any of its arguments. The `statistics` argument must be assigned an instance of the class `TransformationStatistics` as the default value. The `TransformationStatistics` instance must be initialized using the names of the arguments requiring statistics.
+
+!!! warning "Transformation Statistics"
+
+    The statistics provided to the transformation function is the statistics computed using [train set](https://www.hopsworks.ai/dictionary/train-training-set) obtained when training data is generated. It might not portray the statistics of the complete data. Hence, training dataset statistics cannot be obtained for on-demand transformations. 
```

Comment:
[the train set]
Remove this line - it's confusing
It might not portray the statistics of the complete data. 

Training dataset statistics are not available for on-demand transformations. 
(no need to explain why here)